src/main/resources/hapi-r4.properties

```diff
@@ -0,0 +1,151 @@
+# Adjust this to set the version of FHIR supported by this server. See
+# FhirVersionEnum for a list of available constants. Example values include
+# DSTU2, DSTU3, R4.
+fhir_version=R4
+
+# This is the address that the FHIR server will report as its own address.
+# If this server will be deployed (for example) to an internet accessible
+# server, put the DNS name of that server here.
+#
+# Note that this is also the address that the hapi-fhir-testpage-overlay
+# (the web UI similar to the one at http://hapi.fhir.org) will use to
+# connect internally to the FHIR server, so this also needs to be a name
+# accessible from the server itself.
+server_address=http://localhost:8080/hapi-fhir-jpaserver/fhir/
+
+enable_index_missing_fields=false
+auto_create_placeholder_reference_targets=false
+enforce_referential_integrity_on_write=false
+enforce_referential_integrity_on_delete=false
+default_encoding=JSON
+etag_support=ENABLED
+reuse_cached_search_results_millis=60000
+retain_cached_searches_mins=60
+default_page_size=20
+max_page_size=200
+allow_override_default_search_params=true
+allow_contains_searches=true
+allow_multiple_delete=true
+allow_external_references=true
+allow_cascading_deletes=true
+allow_placeholder_references=true
+expunge_enabled=true
+persistence_unit_name=HAPI_PU
+logger.name=fhirtest.access
+logger.format=Path[${servletPath}] Source[${requestHeader.x-forwarded-for}] Operation[${operationType} ${operationName} ${idOrResourceName}] UA[${requestHeader.user-agent}] Params[${requestParameters}] ResponseEncoding[${responseEncodingNoDefault}]
+logger.error_format=ERROR - ${requestVerb} ${requestUrl}
+logger.log_exceptions=true
+datasource.driver=org.h2.Driver
+datasource.url=jdbc:h2:file:./target/database/h2
```

Comment:
Shouldn't we change this to be a MySQL connection?

Reply:
@hankwallace this is the first commit,
I am going to push all the changes needed for this in the next commit.